src/main/java/com/github/zetten/maven/xvfb/AbstractXvfbMojo.java

```diff
@@ -128,17 +136,44 @@
 	protected String fbdir;
 
 	/**
+	 * The number of seconds to wait for return value when stopping Xvfb process.
+	 */
+	@Parameter(defaultValue = "13", required = false)
+	protected long stopTimeoutInSeconds;
```

Comment:
Does this need to be a `long` if it's in seconds? I would say make it an `int`, or change it to milliseconds.

For naming consistency, I think `destroyTimeout` is sufficient (with the unit of time in the javadoc).

Also, please add a user property, e.g. `xfvb.destroy.timeout` so that this can be overridden arbitrarily on the command line.